build/teamcity/internal/release/process/make-and-publish-build-tagging.sh

```diff
@@ -28,51 +37,59 @@ fi
 
 cat << EOF
 
-  build_name:      $build_name
-  release_branch:  $release_branch
-  is_custom_build: $is_custom_build
-  gcr_repository:  $gcr_repository
+  build_name:          $build_name
+  release_branch:      $release_branch
+  is_customized_build: $is_customized_build
+  gcr_repository:      $gcr_repository
```

Comment:
One last question, why not add the `gcs_bucket` to the output?

```suggestion
  gcs_bucket:          $gcs_bucket
  gcr_repository:      $gcr_repository
```

Reply:
I'd say it a cargo cult thing. :D We print the full URLs at the end in any case. This is more like a debugging output IMO. I can add the bucket name if you have a strong opinion.

Reply:
Not a strong opinion. It's good to go. :shipit: